locations/spiders/yaoko_jp.py

```
@@ -0,0 +1,133 @@
+    """
+
+    name = "yaoko_jp"
+    item_attributes = {
+        "brand": "ヤオコー",
+        "brand_wikidata": "Q11344967",
+    }
+
+    start_urls = ["https://yaoko-job.net/jobfind-pc/area/All"]
+    store_list_url = "https://www.yaoko-net.com/store/"
+
+    branch_to_coords_map: dict[str, tuple[float, float]] = {}
```

Ruff (0.16.3)

[warning] Mutable default value for class attribute

(RUF012)